src/Admin/src/Handler/AdminAccountHandler.php

```diff
@@ -24,39 +26,37 @@ class AdminAccountHandler implements RequestHandlerInterface
      * @Inject({
      *     HalResponseFactory::class,
      *     ResourceGenerator::class,
-     *     AdminServiceInterface::class
+     *     AdminServiceInterface::class,
+     *     "config"
      * })
      */
     public function __construct(
         protected HalResponseFactory $responseFactory,
         protected ResourceGenerator $resourceGenerator,
-        protected AdminServiceInterface $adminService
+        protected AdminServiceInterface $adminService,
+        protected array $config,
     ) {
     }
 
     public function get(ServerRequestInterface $request): ResponseInterface
     {
-        return $this->createResponse(
-            $request,
-            $request->getAttribute(Admin::class)
-        );
+        return $this->createResponse($request, $request->getAttribute(Admin::class));
     }
 
+    /**
+     * @throws ConflictException
+     * @throws FormValidationException
+     * @throws NotFoundException
+     */
     public function patch(ServerRequestInterface $request): ResponseInterface
     {
         $inputFilter = (new UpdateAdminInputFilter())->setData($request->getParsedBody());
         if (! $inputFilter->isValid()) {
-            return $this->errorResponse($inputFilter->getMessages());
+            throw (new FormValidationException())->setMessages($inputFilter->getMessages());
```

Comment:
What if we initialize the `FormValidationException` by passing the messages as construtor parameters?
`throw (new FormValidationException($inputFilter->getMessages()))`

Reply:
`$inputFilter->getMessages()` returns a multidimensional associative array.
Unless serializing the errors somehow, I cannot use them to initialize `FormValidationException`'s parent class `Exception`.

That's why I opted for this approach of:
- initializing an empty `FormValidationException`
- setting messages as an array which is later used in `src/App/src/Handler/ResponseTrait.php` when catching `FormValidationException` and returning `$this->errorResponse($exception->getMessages())`.